### ISSUE TITLE
[Blues Brothers] lacking Adlib sound

### ISSUE THREAD

**Issue by nemo93** [dosbox-staging/dosbox-staging#459]
This [game](https://en.wikipedia.org/wiki/The_Blues_Brothers_(video_game)) by Titus does work great yet I'm pretty sure it supported Soundblaster back in the days. The only sound I could get with "Staging" is when I enable the speaker. Manual says Adlib is supported and by setting a Soundblaster (sbpro1/2 or sb16) I should get something other than "beeps". But it's not the case.

Tried with 2 versions of the game (one official, one cracked) and same outcome. There's no setup screen or menu to pick a sound card. Just a basic menu on game's launch asking you to pick a gfx card (ega/cga...).
Tried all flavors of soundcard and mixer settings. Sound will just kick in when `speaker` is set to `true`.

Does anyone successfully got sound other than from speaker? From the many [YT videos](https://www.google.com/search?q=dos+blues+brothers&tbm=vid) out there it seems to be possible.

**Comment by kcgen** [comment]
@nemo93 , excellent find; these reports are absolute gems.

@dreamer, bisecting found that Blues Brothers will plays via Adlib as of cdcd8a4d, but the next merge 053d078a it doesn't detecting it and falls back to PC Speaker.  4353 was all Adlib/OPL code, so that explains it. 
``` 
053d078a   Merge branch 'svn/trunk' r4353 <-- Only plays via PC speaker
73368be1   Revert "Add patch 287 from maccaf (…)" r4351
cdcd8a4d   Merge branch 'svn/trunk' r4350  <--- 
```

**Comment by kcgen** [comment]
Isolated this to SVN 4352 and onward (exclusive of dosbox-staging) and informed The DOSBox Team in SourceForge [Ticket #534](https://sourceforge.net/p/dosbox/bugs/534/).

**Comment by nemo93** [comment]
For the record the following (Titus) games might be affected as well by this "bug": 
* Super Cauldron
* Moktar / Adventures of Moktar (FR version of Titus the Fox which I haven't tried)

**Comment by kcgen** [comment]
Nice hunch @nemo93! confirmed and added to the ticket.

**Comment by dreamer** [comment]
Hmm, there was no activity on this regression in SVN bugtracker for 7 days now.

Since we don't know when (or even if) upstream will fix the issue, I prepared a commit effectively reverting changes from SVN r4352 and r4353 (we don't know what they were supposed to fix, but we know they broke something).

If upstream won't fix this bug in time for our 0.76.0 release, I will apply this commit to master: f9d532d6ba887a18672cada73d0e4e22e33119f9 (it fixes Blues Brothers).

**Comment by dreamer** [comment]
OK, there was no SVN activity upstream for more than a month at this point; I don't think upstream will actually fix this issue. I decided to simply apply the commit reverting the problem: 83546ae571c2979980c9414c78bbff836abfbb33 - tested in Blues Brothers and music is cool again :)
